scripts/data/gen-nodetime/src/relayer/lib/logger.ts

```diff
@@ -7,31 +7,51 @@ interface Logger {
 	verbose: LogMethod;
 	debug: LogMethod;
 }
-
+export enum LogLevels {
+	ERROR = 0,
+	WARN = 1,
+	INFO = 2,
+	VERBOSE = 3,
+	DEBUG = 4
+}
 export default class ConsoleLogger {
 	public readonly error: LogMethod;
 	public readonly warn: LogMethod;
 	public readonly info: LogMethod;
 	public readonly verbose: LogMethod;
 	public readonly debug: LogMethod;
 
-	constructor() {
-		this.error = () => {
+	constructor(logLevel:LogLevels) {
+		this.error = (msg) => {
+			if(logLevel>=0) {
```

Comment:
can we use the `LogLevels` enum instead of the hardcoded numbers?

```suggestion
			if(logLevel>=LogLevels.ERROR) {
```

Reply:
@clockworkgr any updates?

Reply:
woops...somehow missed this comment...apologies...all done now